source/importing-css-assets-and-javascript/index.html.md.erb

```diff
@@ -7,15 +7,36 @@ weight: 70
 
 ## CSS
 
-### Import all the CSS
+To import the CSS from GOV.UK Frontend, you can either:
+
+- add GOV.UK Frontend's CSS file to your HTML
+- import the CSS into your own Sass file
+
+### Add the CSS file to your HTML
+
+If you decide to add the CSS to your HTML, you can do one of the following:
+
+- set up your routing so requests for the CSS file are served from `node_modules/govuk-frontend/dist/govuk/govuk-frontend.min.css`
+- copy the `node_modules/govuk-frontend/dist/govuk/govuk-frontend.min.css` file into your application
+
+Then link the CSS file before the closing `</head>` tag of your HTML page or page template.
+
+```html
+<head>
+  <!-- // ... -->
+  <link rel="stylesheet" href="<YOUR-STYLESHEETS-FOLDER>/govuk-frontend.min.css">
+</head>
```

Comment:
If we go less specific with the previous comment, we should be less specific with the example.

```suggestion
<head>
  <!-- // ... -->
  <link rel="stylesheet" href="<YOUR-STYLESHEETS-FOLDER>/govuk-frontend.min.css">
  <!-- // ... -->
</head>
```